src/ActiveLogin.Authentication.BankId.AspNetCore.QRCoder/ActiveLogin.Authentication.BankId.AspNetCore.QRCoder.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <NeutralLanguage>en</NeutralLanguage>
+    <NoWarn>1701;1702;1591;CS7035</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <AssemblyName>ActiveLogin.Authentication.BankId.AspNetCore.QRCoder</AssemblyName>
```

Comment:
Should it be `ActiveLogin.Authentication.BankId.AspNetCore.QRCoder` or `ActiveLogin.Authentication.BankId.AspNetCore.QrCoder` (casing on R)? Not sure myself.

Reply:
Me neither. Since we are naming the package after the dependency perhaps we should spell it in the same way as the dependency?

Reply:
I agree!